src/libs/auth.js

```diff
@@ -162,7 +162,7 @@ authStore.subscribe(async (state, oldState) => {
   if (!oldState.isSignedIn && state.isSignedIn) {
     window.newrelic.setCustomAttribute('userGoogleId', state.user.id)
     window.Appcues && window.Appcues.identify(state.user.id, {
-      dateJoined: await getDateJoined()
+      dateJoined: state.registrationStatus === 'registered' ? await getDateJoined() : Date.now()
```

Comment:
As written, this will always use `Date.now()` because the registration check doesn't happen until after the sign-in transition. We might want to just delay this action until we see the registration transition.

Reply:
Done.